stories/topics/ref-saas-egress-model.adoc

```diff
@@ -0,0 +1,8 @@
+[id="ref-saas-egress-model"]
+= Egress model or pull connectivity
+
+Remote automation mesh nodes can poll the {PlatformNameShort} without needing to open ingress ports through the firewall, using a 'polling' or 'pull' model. 
+This model will eliminate the need to have hop nodes from your DMZ to private networks if private networks have outbound internet connectivity.
```

Comment:
DMZ here stands for  demilitarized zone which I think is less recognizable than the acronym.